xla/service/gpu/runtime/gpublas_lt_matmul.cc

```diff
@@ -90,7 +90,7 @@ absl::Status DoMatmul(
     std::optional<StridedMemrefView> d_amax, int64_t algorithm,
     double alpha_real, double alpha_imag, double beta,
     DotDimensionNumbers dot_dims, se::gpu::BlasLt::Epilogue epilogue,
-    absl::Span<const int32_t> precision) {
+    absl::Span<const int32_t> precision, bool grad_x, bool grad_y) {
```

Comment:
Can we have a better name for grad_x and grad_y? I see that they are bools but it's not clear what they represent? Is it "grad_x_needed"? Or "have_grad_x"?

Reply:
perhaps, naming is not the best.. but I was trying to keep it consistent with the rest of the code (these flags are propagated from [tensorflow](https://github.com/tensorflow/tensorflow/pull/62362/files#diff-18efe08e84012acb9178006d37cff2cfddf2a6411d75b37de8a4cb1c70313c36L1666-R1692)) 

grad_x = true means that we are computing gradient of the first operand (i.e., back propagation for input 1)
grad_y = true means that we are computing gradient of the second operand (i.e., back propagation for input 2)

Reply:
ok thanks